core/controllers/galleries.py

```diff
@@ -123,29 +167,23 @@ class GalleryHandler(base.BaseHandler):
 
     def get(self):
         """Handles GET requests."""
-        # TODO(sll): Figure out what to do about explorations in categories
-        # other than those explicitly listed.
-
         query_string = self.request.get('q')
```

Comment:
Unescaping is unneeded as it used to work previously as is. If possible, it's probably better to discuss further this on chat.